main.nf

```diff
@@ -495,12 +495,50 @@ process annotate_rsid {
     """
 }
 
-// Filter variants, adding various filter tags to the "FILTER" field of the VCF.
-// FIXME: I'm getting warnings that MQRankSum and ReadPosRankSum don't exist.
-process filter_variants {
+
+// Splitting VCF into SNPs and indels, because they have to be filtered seperately
+process subset_SNPs {
+
+    input:
+    set sample, file(vcf), file(idx) from rsid_annotated_vcf_ch
+
+    output:
+    set sample, file("snp.vcf"), file("snp.vcf.idx") into snpsubset_filter_ch
+
+    script:
+    """
+    gatk SelectVariants \
+    -V $vcf \
+    -select-type SNP \
+    -O "snp.vcf"
+    --java-options "-Xmx${task.memory.toGiga()}g -Xms${task.memory.toGiga()}g"
+    """
+}
+
+process subset_indels {
+
     input:
     set sample, file(vcf), file(idx) from rsid_annotated_vcf_ch
 
+    output:
+    set sample, file("indel.vcf"), file("indel.vcf.idx") into indelsubset_filter_ch
+
+    script:
+    """
+    gatk SelectVariants \
+    -V $vcf \
+    -select-type INDEL \
+    -O "indel.vcf"
+    --java-options "-Xmx${task.memory.toGiga()}g -Xms${task.memory.toGiga()}g"
+    """
+}
+
+// Hard filter variants, adding various filter tags to the "FILTER" field of the VCF.
+// FIXME: I'm getting warnings that MQRankSum and ReadPosRankSum don't exist.
+process hard_filter_SNPs {
```

Comment:
@ElisabetThomsen Tað er betur vissi `hard_filter_SNPs` og `HardFilterINDELs` navngevingin líkist.

Reply:
I have fixed this 👍